account-kit/rn-signer/ios/turnkey/AuthManager.swift

```diff
@@ -0,0 +1,85 @@
+//
```

Comment:
Do we need these once we have turnkey in pods? Did we copy these files or did we build them directly?